src/admin/AdminDashboard.js

```diff
@@ -10,9 +10,9 @@ import ListItemIcon from '@mui/material/ListItemIcon';
 import ListItemText from '@mui/material/ListItemText';
 import HomeIcon from '@mui/icons-material/Home';
 import PersonIcon from '@mui/icons-material/Person';
-// import BloodtypeIcon from '@mui/icons-material/Bloodtype';
-// import MenuBookIcon from '@mui/icons-material/MenuBook';
-// import FolderOpenIcon from '@mui/icons-material/FolderOpen';
+import BloodtypeIcon from '@mui/icons-material/Bloodtype';
```

Comment:
These aren't yet connected I don't think. Like the links still show ice cream and yoghurt. Mostly up to you if you want to merge it like that. I'll request changes so you can take a look at the comment.